### PR TITLE
react-1271-skill-map-graph-hover-tooltip

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
  Skill Map Graph - hover Skill Map Period
[#1271](https://github.com/nerubia/kh360-react/issues/1271)
  
 BE: https://github.com/nerubia/kh360-nodejs/pull/695

### CODE REVIEW

File: src/custom-types/skill-map-search-type.ts [hunk 1]
```diff
@@ -60,3 +60,7 @@ export const SkillMapSearchSortNames = {
   [SkillMapSearchSortOptions.NAME]: "Name",
   [SkillMapSearchSortOptions.SKILL]: "Skill",
 }
+
+export interface CustomeSkillMapDetails {
```
Comment (jatnerubia) [inline src/custom-types/skill-map-search-type.ts:64]: typo

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a minor change ^^

File: src/custom-types/skill-map-search-type.ts (hunk 1 above)
Comment (chadiegilnerubia) [reply src/custom-types/skill-map-search-type.ts]: done :3

**Review by jatnerubia — APPROVED** [review]
Cool

**Pull request merged**